modules/gitops-configuring-sscsi-driver-to-mount-secrets-from-aws-secrets-manager.adoc

```diff
@@ -0,0 +1,186 @@
+----
+
+.. Create a YAML file with the following configuration for the credentials request in the `/environments/dev/credentialsrequest-dir-aws/` path to mount a secret to the deployment pod in the `dev` namespace:
++
+.Example `credentialsrequest.yaml` file
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: aws-provider-test
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - "secretsmanager:GetSecretValue"
+      - "secretsmanager:DescribeSecret"
+      effect: Allow
+      resource: "<aws_secret_arn>" # <1>
+secretRef:
+  name: aws-creds
+  namespace: dev 
+serviceAccountNames:
+  - default
+----
+<1> The ARN of your secret in the region where your cluster is on. The `<aws_region>` of `<aws_secret_arn>` has to match the cluster region. If it does not match, create a replication of your secret in the region where your cluster is on. 
++
+[TIP]
+====
+To find your cluster region, run the command:
+
+[source,terminal]
+----
+$ oc get infrastructure cluster -o jsonpath='{.status.platformStatus.aws.region}'
+----
+
+.Example output
+[source,terminal]
+----
+us-west-2
+----
+====
+
+.. Retrieve the OIDC provider by running the following command:
++
+[source,terminal]
+----
+$ oc get --raw=/.well-known/openid-configuration | jq -r '.issuer'
+----
++
+.Example output
+[source,terminal]
+----
+https://<oidc_provider_name>
+----
+Copy the OIDC provider name `<oidc_provider_name>` from the output to use in the next step.
+
+.. Use the `ccoctl` tool to process the credentials request by running the following command:
++
+[source,terminal]
+----
+$ ccoctl aws create-iam-roles \
+    --name my-role --region=<aws_region> \
+    --credentials-requests-dir=credentialsrequest-dir-aws \
+    --identity-provider-arn arn:aws:iam::<aws_account>:oidc-provider/<oidc_provider_name> --output-dir=credrequests-ccoctl-output
+----
++
+.Example output
+[source,terminal]
+----
+2023/05/15 18:10:34 Role arn:aws:iam::<aws_account_id>:role/my-role-my-namespace-aws-creds created
+2023/05/15 18:10:34 Saved credentials configuration to: credrequests-ccoctl-output/manifests/my-namespace-aws-creds-credentials.yaml
+2023/05/15 18:10:35 Updated Role policy for Role my-role-my-namespace-aws-creds
+----
++
+Copy the `<aws_role_arn>` from the output to use in the next step. For example, `arn:aws:iam::<aws_account_id>:role/my-role-my-namespace-aws-creds`.
+
+.. Check the role policy on AWS to confirm the <aws_region> of `"Resource"` in the role policy matches the cluster region:
++
+.Example
+[source,json]
+----
+{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Effect": "Allow",
+			"Action": [
+				"secretsmanager:GetSecretValue",
+				"secretsmanager:DescribeSecret"
+			],
+			"Resource": "arn:aws:secretsmanager:<aws_region>:<aws_account_id>:secret:your-secret-xxxxxx"
```

Comment:
Let's keep the naming consist. For example we have `my-namespace`, `your-domain`, `your-secret-` etc in same documentation which doesn't look good.